src/main/react-front-end/src/MainComponent.js

```diff
@@ -48,6 +50,7 @@ export default class MainComponent extends Component {
 
         <div style={{ marginTop: '50px', display: 'block' }}>
           <Switch>
+            <Route path='/account' component={AccountControlComponent}/>
```

Comment:
Removing the AccountControlComponent from App.js and adding it in the MainComponent.js added Navbar to Sign page and register page.

Reply:
Understood now